perf/README.md

```diff
@@ -60,3 +60,37 @@ $ make
 
 This will *not upload* the results to the dashboard, as it is reserved
 for CI jobs which run more or less in a consistent environment.
+
+Troubleshooting CI with the semgrep-dev Docker image
```

Comment:
It might be useful to previously state that we run these benchmarks nightly, and give the path to the CircleCI config file, since we have the docker troubleshooting info

Reply:
I didn't mention specifics about CI because (1) this is independent from the CI setup and (2) we will forget to update this document when the schedule or the nature of the jobs changes.

Reply:
I added a generic note about where to find the CI config.